Dockerfile

```diff
@@ -1,11 +1,26 @@
 # Running from node
 from node:lts-alpine3.11
 
-run apk add python make g++ ffmpeg
+# Set time zone - default of America/Phoenix
+ARG TZ='America/Phoenix'
+ENV DEFAULT_TZ ${TZ}
```

Comment:
I'd prefer to see this a one line. Although these won't create layers and increase the size of the build; I believe both of these commands still add intermediate build images.

Reply:
They have to be separate if we want to be able to configure the time zone at image build time. The ARG is just setting the default value, but that value can be overriden:

docker build --build-arg TZ=America/Chicago .

or in a docker-compose file:

```
build:
  context: .
  args:
    TZ: America/Phoenix
```

Reply:
https://vsupalov.com/docker-build-pass-environment-variables/